FirebaseFunctions/Sources/HTTPSCallable.swift

```diff
@@ -82,16 +91,26 @@ open class HTTPSCallable: NSObject {
   @objc(callWithObject:completion:) open func call(_ data: Any? = nil,
                                                    completion: @escaping (HTTPSCallableResult?,
                                                                           Error?) -> Void) {
-    functions.callFunction(name: name,
-                           withObject: data,
-                           timeout: timeoutInterval) { result in
+    let callback: ((Result<HTTPSCallableResult, Error>) -> Void) = { result in
       switch result {
       case let .success(callableResult):
         completion(callableResult, nil)
       case let .failure(error):
         completion(nil, error)
       }
     }
+
+    if let name = name {
+      functions.callFunction(name: name,
+                             withObject: data,
+                             timeout: timeoutInterval,
+                             completion: callback)
+    } else {
```

Comment:
`else if let url = url`

I don't have a strong opinion about generating an error in a final `else` or ignoring it.